src/masonite/validation/Validator.py

```diff
@@ -1029,6 +1029,128 @@ def negated_message(self, attribute):
         return "The {} is a valid {} postal code.".format(attribute, self.locale)
 
 
+class different(BaseValidation):
+    """The field under validation must be different than an other given field."""
+    def __init__(self, validations, other_field, messages={}, raises={}):
+        super().__init__(validations, messages=messages, raises=raises)
+        self.other_field = other_field
+
+    def passes(self, attribute, key, dictionary):
+        other_value = dictionary.get(self.other_field, None)
+        return attribute != other_value
+
+    def message(self, attribute):
+        return "The {} value must be different than {} value.".format(attribute, self.other_field)
+
+    def negated_message(self, attribute):
+        return "The {} value be the same as {} value.".format(attribute, self.other_field)
+
+
+class uuid(BaseValidation):
+    """The field under validation must be a valid UUID. The UUID version standard
+    can be precised (1,3,4,5)."""
+    def __init__(self, validations, version=None, messages={}, raises={}):
```

Comment:
Let's have a sensible default here. I would say the most common in version 4

Reply:
Yes UUID4 is the most widely used. I also updated the PR for the docs.